apps/api/src/app/models/userProfiles/user-profile.interface.ts

```diff
@@ -1,12 +1,18 @@
 import { Document } from 'mongoose';
 
 export interface CreateUserProfile {
+  type: string;
+  description: string;
   firstName: string;
   lastName: string;
   photo: string;
   fieldOfProfession: string;
+  company?: string;
   categories: string[];
   city: string;
+  phone: string;
+  email: string;
+  socialLinks?: string[];
 }
 export interface UserProfile {
```

Comment:
way Create payload is bigger than User interface? I think it should be biggest priority interface